pkg/tester/construction.go

```diff
@@ -136,6 +136,12 @@ func InitializeConstruction(
 		fetcher.WithTimeout(time.Duration(config.HTTPTimeout)*time.Second),
 	)
 
+	// Import prefunded account and save to database
+	err = keyStorage.ImportAccounts(ctx, config.Construction.PrefundedAccounts)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Do you want to write a TODO here for balance loading? I'd recommend doing that in a separate PR